algorithms.py

```diff
@@ -566,6 +566,22 @@ def migration_event(self, j, k):
             u = u.next
         # print("AFTER Population sizes:", [len(pop) for pop in self.P])
 
+    def store_edges_left(self, y, u):
+        self.store_edge(y.left, y.right, u, y.node)
+        y.node = u
+        while y.prev is not None:
+            y = y.prev
+            self.store_edge(y.left, y.right, u, y.node)
```

Comment:
This will do the same thing, I think, and is simpler:
```python
while y is not None:
    self.store_edge(y.left, y.right, u, y.node)
    y.node = u
    y = y.prev
```

Reply:
You're right. I'll tweak that, and fix the whitespace that CircleCI is complaining about.